includes/Modules/Analytics_4.php

```diff
@@ -389,13 +457,42 @@ private function handle_provisioning_callback( $account_id ) {
 	 * @return RequestInterface|callable|WP_Error Request object or callable on success, or WP_Error on failure.
 	 *
 	 * @throws Invalid_Datapoint_Exception Thrown if the datapoint does not exist.
+	 * @throws Missing_Required_Param_Exception Thrown if a required parameter is missing or empty.
+	 * phpcs:ignore Squiz.Commenting.FunctionCommentThrowTag.WrongNumber
 	 */
 	protected function create_data_request( Data_Request $data ) {
 		switch ( "{$data->method}:{$data->datapoint}" ) {
 			case 'GET:accounts':
 				return $this->get_service( 'analyticsadmin' )->accounts->listAccounts();
 			case 'GET:account-summaries':
 				return $this->get_service( 'analyticsadmin' )->accountSummaries->listAccountSummaries( array( 'pageSize' => 200 ) );
+			case 'POST:create-account-ticket':
+				if ( empty( $data['displayName'] ) ) {
+					throw new Missing_Required_Param_Exception( 'displayName' );
+				}
+				if ( empty( $data['regionCode'] ) ) {
+					throw new Missing_Required_Param_Exception( 'regionCode' );
+				}
+				if ( empty( $data['propertyName'] ) ) {
+					throw new Missing_Required_Param_Exception( 'propertyName' );
+				}
+				if ( empty( $data['dataStreamName'] ) ) {
+					throw new Missing_Required_Param_Exception( 'dataStreamName' );
+				}
```

Comment:
Right, it isn't possible right now because `datapoint` requests are dynamic (route parameters), so we can't register different schemas for each without registering them independently, but that would be a nice enhancement to make in the future 👍

Reply:
Sounds like a good future update - I've created #6759 to track this.